src/_locales/en/messages.json

```diff
@@ -195,11 +195,11 @@
     "set_up": "Set up",
     "settings": "Settings",
     "settings_disable_new_background_tab": "Disable open hidden tabs in the background",
-    "settings_disable_new_background_tab_desc": "This is an deprecated behavior of Maskbook. If you find that you can not decrypt some old posts, deselect this.",
+    "settings_disable_new_background_tab_desc": "This is a deprecated behavior of Maskbook. If you find that you cannot decrypt some old posts, deselect this.",
```

Comment:
I advised that all wordings should be in the positive tone unless it is a button for an action like "unsubscribe". If there is a switch for email notification, it should be "Receive email notifications" instead of "Disable email notifications". Imagine when you enable an option which disables something. Therefore, it will be better to write "Deprecated behavior. Enable if you need to decrypt ancient posts (created before Jan 2020)." and make its default value `false`, where the user can enable this option to enable the behavior.